<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Changed</class>
 <widget class="QWidget" name="Changed">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>495</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>修改界面</string>
  </property>
  <widget class="QPushButton" name="returnpushButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>450</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>主菜单</string>
   </property>
  </widget>
  <widget class="QLabel" name="namelabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>姓名</string>
   </property>
  </widget>
  <widget class="QLabel" name="relalabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>140</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>关系</string>
   </property>
  </widget>
  <widget class="QLabel" name="marklabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>200</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>备注</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="namelineEdit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>90</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="relalineEdit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>140</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="marklineEdit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>200</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="numberlabel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>370</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>手机号码</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="numberlineEdit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>370</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="delepushButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>450</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="changedpushButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>450</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
